client/src/i18n/fr/stock.json

```diff
@@ -29,8 +29,6 @@
     "ENTRY_DONATION"  : "Donation",
     "ENTRY_INTEGRATION" : "Intégration",
     "ENTRY_TRANSFER"  : "Transfert",
-    "EXCLUDE_ALL_EXAUSTED_STOCK": "Exclure tous les stocks épuisés",
```

Comment:
Are you sure these aren't used anywhere?

Reply:
I could not find these in global searches.  So I think they are not used at all.